laplace/curvature/curvature.py

```diff
@@ -39,13 +40,18 @@ def __init__(self, model, likelihood, last_layer=False, subnetwork_indices=None)
         else:
             self.lossfunc = CrossEntropyLoss(reduction='sum')
             self.factor = 1.
+        self.params = [p for p in self._model.parameters() if p.requires_grad]
+        self.params_dict = {k: v for k, v in self._model.named_parameters() if v.requires_grad}
+        self.buffers_dict = {k: v for k, v in self.model.named_buffers()}
 
     @property
     def _model(self):
         return self.model.last_layer if self.last_layer else self.model
 
     def jacobians(self, x, enable_backprop=False):
-        """Compute Jacobians \\(\\nabla_\\theta f(x;\\theta)\\) at current parameter \\(\\theta\\).
+        """
```

Comment:
Comment should start right after `"""` as in all docstrings. What's the reason for the deviation here?